CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 * Update `output.md` and add in 'Pipeline information' section describing standard NF and pipeline reporting.
 * Build Docker image using GitHub Actions, then push to Docker Hub (instead of building on Docker Hub)
 * New Slack channel badge in pipeline readme
+* Add AWS tests GitHub Actions workflow
```

Comment:
Either make clear which one is which if there is a difference, or merge into a single line :-)